client/components/notice/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { Notice } from '../index';
+
+describe( 'index', function() {
```

Comment:
Ah, excellent point. Provides better context when looking at the test output. Changed it to `Notice`.